src/components/Result/ResultHeader.vue

```diff
@@ -14,9 +14,14 @@
 				>
 					{{ title }}
 				</p>
-				<p class="hidden md:flex text-xl">
+				<p v-if="description !== ''" class="hidden md:flex text-xl">
 					{{ description }}
 				</p>
+				<p v-if="documents.length === 2" class="hidden md:flex text-xl md:invisible ">
```

Comment:
For this, you could use component slots. You would just pass this html code only in component you need it, and you don't have to mix functionality here (e.g. passing documents as a prop here and other stuff.)
https://vuejs.org/v2/guide/components-slots.html